packages/ui-transfer-components/lib/components/loading.component.ts

```diff
@@ -16,6 +19,7 @@ export class LoadingComponent extends RxLitElement {
     return html`
       <div class="loading-container">
         ${unsafeSVG(Loading)}
+        <p>${this.message ? this.message : ''}</p>
       </div>
     `;
```

Comment:
If there is no message, there should be no paragraph tags. Also, you should be able to just let LitElement process the undefined properties, which is i.m.o. cleaner than the empty string (even though it has the same result).
```suggestion
    const msg = this.message ? html`<p>this.message</p>` : undefined

    return html`
      <div class="loading-container">
        ${unsafeSVG(Loading)}
        ${msg}
      </div>
    `;
```

Reply:
This was already fixed after comment by @BelgianNoise